src/patternfly/components/Spinner/spinner.scss

```diff
@@ -0,0 +1,210 @@
+// Spinner Component
+
+.pf-c-spinner {
+  --pf-c-spinner--AnimationDuration: 1.5s;
+  --pf-c-spinner--AnimationTimingFunction: cubic-bezier(.77, .005, .315, 1);
+
+  // This conceptual variable is used to set other values all based on the diameter of the spinner
+  --pf-c-spinner--diameter: var(--pf-global--icon--FontSize--xl);
```

Comment:
Should the spinner have a default size in core? Or should we always require use with a modifier, and we can offer a default size in react? If we keep the default size in core, I think we should mention in the documentation the default is `pf-m-xl`.

Reply:
+1 it seems confusing to have a modifier that does nothing, otherwise we should remove the default and require a modifier. I don't feel strongly about either, I guess It could depend on how we've handled this in other places.